content/en/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes.md

```diff
@@ -396,10 +396,13 @@ have a number of fields that you can use to more precisely control the behavior
 liveness and readiness checks:
 
 * `initialDelaySeconds`: Number of seconds after the container has started before startup,
-  liveness or readiness probes are initiated. If a startup  probe is defined, liveness and
-  readiness probe delays do not begin until the startup probe has succeeded. If the value of
-  `periodSeconds` is greater than `initialDelaySeconds` then the `initialDelaySeconds` will be
-  ignored. Defaults to 0 seconds. Minimum value is 0.
+  liveness or readiness probes are initiated. If a startup probe is defined, liveness and
+  readiness probe delays do not begin until the startup probe has succeeded.
+  An `initialDelaySeconds` value greater than 0 will delay when the first probe starts. If
+  the value of `periodSeconds` is greater than `initialDelaySeconds` then `periodSeconds`
+  intervals remain effective. For example, if `initialDelaySeconds` is set to 20, but
+  `periodSeconds` is set to 10, periodic probes will still begin after an initial 20 seconds.
+  Defaults to 0 seconds. Minimum value is 0.
```

Comment:
The logic is much more confusing in the revised text.

> If the value of `periodSeconds` is greater than `initialDelaySeconds` then `periodSeconds` intervals remain effective.

This talks about `periodSeconds > initialDelaySeconds`.

> For example, if `initialDelaySeconds` is set to 20 ...

This talks about `initialDelaySeconds > periodSeconds`. It is not an example of the
sentence above.

Please clear your mind first and then try articulate what you are trying to clarify here.

Reply:
Sorry that was a typo, I stick to the discussions above, and have modified the text.

Reply:
Alright. The revised text makes sense now.
The next question then is what are the value of adding this example.
The existing text already indicated that the delay threshold is ignored when the period is longer than delay, right?

Reply:
The intention of the PR was to remove the incorrect explanation "_If the value of `periodSeconds` is greater than `initialDelaySeconds` then the `initialDelaySeconds` will be ignored_", and then advised to precisely describe the actual system behavior, but it doesn't seems easy to come up with a good explanation. If no one knows how to explain it in a better way, should we just remove the wrong explanation, like the first version of the PR does: https://github.com/kubernetes/website/pull/48536/commits/78c4a52a05f12beb33e47f7be169e99aed7ae2da

Reply:
When periodSeconds>initialDelaySeconds, the initialDelaySeconds is ignored. Isn't that what you are trying to clarify here? The existing text is clear on that, right?

Reply:
The point of the PR is that's wrong. I thought this has been clarified in https://github.com/kubernetes/website/pull/48536#issuecomment-2442786681 and the issue https://github.com/kubernetes/website/issues/48519?